src/main/java/io/ebean/config/PlatformConfig.java

```diff
@@ -0,0 +1,282 @@
+package io.ebean.config;
+
+import java.util.ArrayList;
+import java.util.List;
+import java.util.Map;
+import java.util.Map.Entry;
+
+
+import io.ebean.annotation.Platform;
+import io.ebean.config.dbplatform.DbType;
+import io.ebean.config.dbplatform.IdType;
+import io.ebean.util.StringHelper;
+
+/**
+ * Configuration for Platforms such as UUID, Geometry etc.
+ */
+public class PlatformConfig {
```

Comment:
So this is a rename plus adds allQuotedIdentifiers and databaseSequenceBatchSize ...

Reply:
And ultimately this is to support ... a single ServerConfig with many platforms for migration, so IdType needs to move from ServerConfig to here.